### ISSUE TITLE
Can I run two signal servers in one operating system?

### ISSUE THREAD

**Issue by modder2** [andyet/signalmaster#50]
Can I run two signal servers (one for production version and one for development version) in one server (operating system)?

**Comment by fippo** [comment]
running them on different ports (config.server.port) is probably the easiest way to accomplish this

**Comment by modder2** [comment]
@fippo One command to run is:

    $ NODE_ENV=production node prod/server.js

And other is:

    $ NODE_ENV=development node dev/server.js

Right?

**Comment by fippo** [comment]
yes. You would have different ports in config/development.json and config/production.json respectively
